### ISSUE TITLE
hello,will there be pre-trained models?

### ISSUE THREAD

**Issue by ajacky4504** [JusperLee/Looking-to-Listen-at-the-Cocktail-Party#1]

**Comment by JusperLee** [comment]
my pretrained model don’t be saved




| |
李凯
|
|
邮箱：likai614020758@126.com
|

签名由 网易邮箱大师 定制

**Comment by ajacky4504** [comment]
你好，请问一下可以跟你交换微信嘛，我在训练模型上碰上一些问题 想和你请教，我目前研究领域和语音分离有关。

Kai Li <notifications@github.com> 於 2019年12月20日 週五 下午4:47寫道：

**Comment by JusperLee** [comment]
sorry I am not saving the pretrained model

**Comment by JusperLee** [comment]
You can send your wechat number via email

**Comment by JusperLee** [comment]
I am also a beginner in the field of speech and learn from each other.  My email: tsinghua.kaili@gmail.com. @ajacky4504

**Comment by ajacky4504** [comment]
[image: 2019-12-20 16-55-36 的螢幕擷圖.png]
你好 我在使用python3 audio_data.py   跑出這樣的錯誤 github上面的code來執行
但是更改過下面圖片裡面data range=(0,19)就可以運行 可以請問為什麼嘛
[image: 2019-12-20 17-00-34 的螢幕擷圖.png]

**Comment by JusperLee** [comment]
The audio_data script is my code for processing audio data. 0-19 represents a total of 20 video numbers with the largest number being 19.  But if you exceed 19 when downloading a video, you need to correspond to the largest video label.

**Comment by ajacky4504** [comment]
請問下 從github下載下來 原本是(0,20) 市需要自己手動更改嘛？

Kai Li <notifications@github.com> 於 2019年12月20日 週五 下午5:05寫道：

**Comment by JusperLee** [comment]
0-20 represents the order of video links in the csv file.  If it is 0-5000, it means that I want to download 5000 videos in order.

**Comment by ajacky4504** [comment]
請問下如果設定(0,20)   valid_face_text 是不是必須也要20個呢
因為我照Dataset下載 0到19個圖片中 其中2和12會判定不能用 總共只有18個 才導致錯誤嘛


Kai Li <notifications@github.com> 於 2019年12月20日 週五 下午5:22寫道：

**Comment by ajacky4504** [comment]
[image: 2019-12-20 17-33-00 的螢幕擷圖.png]

莊書愷 <ajacky4504@gmail.com> 於 2019年12月20日 週五 下午5:30寫道：

**Comment by ajacky4504** [comment]
ata_range = (0, 19)  # data usage to generate database   請問下這個參數
市根據下載的音檔數量(假設我下載20部）還是 根據valid_face_text(裡面的數量）
audio_norm_path = os.path.expanduser("./norm_audio_train")
database_path = '../AV_model_database'
frame_valid_path = '../video_data/valid_face_text.txt'
num_speakers = 2
max_generate_data = 50

莊書愷 <ajacky4504@gmail.com> 於 2019年12月20日 週五 下午5:33寫道：

**Comment by JusperLee** [comment]
According to vaild_face_text

**Comment by ajacky4504** [comment]
[image: 2019-12-20 17-44-12 的螢幕擷圖.png]
谢谢这个问题排除了
请问 我做完github前处李步骤但没有跑出AVdataset_train.txt  还是是指dataset_train.txt.

Kai Li <notifications@github.com> 於 2019年12月20日 週五 下午5:38寫道：

**Comment by ajacky4504** [comment]
我在 audio_data.py中 在 213行
with open('%s/dataset_train.txt' % database_repo, 'a') as f:
for line in train:
f.write(line) f.write('\n') 请问这裡是不是和train.py中的59行
with open((database_path + 'AVdataset_train.txt'), 'r') as t: trainfile =
t.readlines() 有出入呢


莊書愷 <ajacky4504@gmail.com> 於 2019年12月20日 週五 下午5:46寫道：

**Comment by ajacky4504** [comment]
這是我的微信 akaiii841119

莊書愷 <ajacky4504@gmail.com>於 2019年12月20日 週五，下午5:54寫道：
